optuna/samplers/_base.py

```diff
@@ -193,3 +195,33 @@ def _raise_error_if_multi_objective(self, study: Study) -> None:
                 "If the study is being used for multi-objective optimization, "
                 f"{self.__class__.__name__} cannot be used."
             )
+
+
+_CONSTRAINTS_KEY = "sampler:constraints"
```

Comment:
Now, I feel the meaning of "sampler" is not clear. How about just using "constraints"?

```suggestion
_CONSTRAINTS_KEY = "constraints"
```

Reply:
This is because this key is only used in samplers (not pruners or importance evaluators.) What do you think?

Reply:
I think it can be better to just say `constraints` in case we need the information of constraints in e.g. importance evaluators.

Reply:
I'll apply your suggestion.